go/src/koding/kites/kloud/metadata/metadata.go

```diff
@@ -1,11 +1,27 @@
 package metadata
 
-import "text/template"
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"koding/kites/config"
+)
 
 //go:generate $GOPATH/bin/go-bindata -mode 420 -modtime 1470666525 -pkg metadata -o provision.sh.go provision.sh
 //go:generate gofmt -l -w -s provision.sh.go
 
-var provision = mustTemplate("provision.sh")
+// Default asset URLs used by kloud when deploying klient.
+//
+// They are being downloaded on a remote vm during provisioning.
+//
+// TODO(rjeczalik): make it configurable during ./configure = add to kites/config/config.json
+var (
+	DefaultEntrypointBaseURL = "https://koding-klient.s3.amazonaws.com/entrypoint"
```

Comment:
we could utilize koding.com/s3 endpoint  here or we could copy entrypoint to dl bucket and serve from there from now on